src/addrman.h

```diff
@@ -229,13 +229,48 @@ friend class CAddrManTest;
     //! Holds addrs inserted into tried table that collide with existing entries. Test-before-evict discipline used to resolve these collisions.
     std::set<int> m_tried_collisions;
 
+    void CheckWithLockHeld() EXCLUSIVE_LOCKS_REQUIRED(cs)
```

Comment:
Make this (and other `Check()` functions) `const`?

Reply:
This change is not trivial due to the non-const [`std::map::operator[]`](https://en.cppreference.com/w/cpp/container/map/operator_at):
```
addrman.cpp:433:20: error: no viable overloaded operator[] for type 'const std::map<CNetAddr, int>'
        if (mapAddr[info] != n)
            ~~~~~~~^~~~~
```